src/components/authentication/Login.jsx

```diff
@@ -29,6 +33,10 @@ export const Login = () => {
           type: LOG_IN_SUCCESS,
           payload: { userToken: encodedToken, userData: foundUser },
         });
+        wishListDispatch({
+          type: INITIALIZE_WISH_COUNT,
+          totalProductsInwishList: foundUser.wishlist.length,
+        });
         navigate(-1);
```

Comment:
What does `navigate(-1)` do?

Reply:
+1

Reply:
navigate(-1) is equivalent to hitting the back button.
- when user click on login, user will get navigated to login page
Use case
1. when user click on login from product list page, after successful login user will get navigated back to product list page
2. when user click on login from home page, after successful login user will get navigated back to home page.

https://reactrouterdotcom.fly.dev/docs/en/v6/api#usenavigate

Reply:
Wow, this is great! Thanks for a clear explanation 🤝